apps/freeablo/faworld/potion.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+#include "item.h"
+#include "player.h"
+
+namespace FAWorld
+{
+    // Class Potion for all potion functions
+    class Potion
+    {
+    public:
+        Potion();
+        ~Potion();
```

Comment:
This class doesn't need to have a constuctor or destructor, the methods could all be made static, or it could turned into a namespace instead.